sunpy/map/compositemap.py

```diff
@@ -460,6 +500,8 @@ def plot(self, axes=None, annotate=True, # pylint: disable=W0613
                 "norm": m.plot_settings['norm'],
                 "alpha": m.alpha,
                 "zorder": m.zorder,
+                "vmin":m.vmin,
+                "vmax":m.vmax,
```

Comment:
@sudk1896 [Travis is complaining](https://travis-ci.org/sunpy/sunpy/jobs/152619855#L1835) in these lines! Check why not the `AIAMap` mentioned in that error and used to build the documentation says that doesn't have such attribute.